src/main/resources/hudson/plugins/plot/XMLSeries/config.jelly

```diff
@@ -0,0 +1,54 @@
+<?jelly escape-by-default='true'?>
+<j:jelly xmlns:j="jelly:core" xmlns:st="jelly:stapler" xmlns:d="jelly:define" xmlns:l="/lib/layout" xmlns:t="/lib/hudson" xmlns:f="/lib/form">
+    <f:entry>
+    <table width="100%" bgcolor="#EEEEEE">
+    <f:entry title="${%Data series file}" help="/plugin/plot/help-series.html">
+        <f:textbox name="file" value="${series.file}"
+                   checkUrl="'descriptorByName/hudson.plugins.plot.pipeline.PlotPublisher/checkSeriesFile?value='+toValue(this)" />
+    </f:entry>
+        <f:nested>
+            <div>
+                <table>
+                    <tr>
+                        <td align="center">${%XPath Result type:}</td>
+                        <th align="center">${%Nodeset}</th>
+                        <th align="center">${%Node}</th>
+                        <th align="center">${%String}</th>
+                        <th align="center">${%Boolean}</th>
+                        <th align="center">${%Number}</th>
+                    </tr>
+                    <tr>
```

Comment:
I think using the f:select jelly form control with a BoxModel is more idiomatic -- would have to manually test to see if this behaves correctly, but it seems like you would want to use Jelly conditional logic to only show the final type rather than an array of checkboxes (cleaner UI).